tests/security-test.py

```diff
@@ -0,0 +1,79 @@
+#!/usr/bin/env python
+import logging
+import unittest
+import yaml
+
+from framework.prototype import (
+    RayClusterAddCREvent
+)
+
+from framework.utils import (
+    shell_subprocess_run,
+    CONST,
+    K8S_CLUSTER_MANAGER,
+    OperatorManager
+)
+
+logger = logging.getLogger(__name__)
+logging.basicConfig(
+    format='%(asctime)s,%(msecs)d %(levelname)-8s [%(filename)s:%(lineno)d] %(message)s',
+    datefmt='%Y-%m-%d:%H:%M:%S',
+    level=logging.INFO
+)
+
+class PodSecurityTestCase(unittest.TestCase):
+    """Test for the document for the Pod security standard in CI """
+    def test_pod_security(self):
+        '''
+        Create a RayCluster with proper securityContext configurations in a namespace 
+        that force the restricted Pod security standard to all pods.
+        '''
+        cluster_with_pod_security = CONST.REPO_ROOT.joinpath("ray-operator/config/security/ray-cluster.pod-security.yaml")
+        cluster_without_pod_security = CONST.REPO_ROOT.joinpath("ray-operator/config/samples/ray-cluster.complete.yaml")
+        kind_config = CONST.REPO_ROOT.joinpath("ray-operator/config/security/kind-config.yaml")
+        image_dict = {
+            CONST.RAY_IMAGE_KEY: 'rayproject/ray:2.2.0',
```

Comment:
https://github.com/ray-project/kuberay/blob/1d3f53721de17f6d5fa0910643c0c5909d32c0dd/ray-operator/config/security/ray-cluster.pod-security.yaml#L29

`ray-cluster.pod-security.yaml` uses `rayproject/ray-ml:2.2.0` rather than `rayproject/ray:2.2.0`.

Reply:
We need to retrieve images from YAML files automatically in the future.